examples/classification/eval_imagenet.py

```diff
@@ -28,11 +29,10 @@ def main():
         description='Learning convnet from ILSVRC2012 dataset')
     parser.add_argument('val', help='Path to root of the validation dataset')
     parser.add_argument(
-        '--model', choices=(
-            'vgg16',
-            'resnet50', 'resnet101', 'resnet152',
-            'se-resnet50', 'se-resnet101', 'se-resnet152',
-            'se-resnext50', 'se-resnext101'))
+        '--model',
+        choices=('vgg16', 'resnet50', 'resnet101', 'resnet152', 'se-resnet50',
+                 'se-resnet101', 'se-resnet152', 'se-resnext50',
+                 'se-resnext101', 'mobilenetv2'))
```

Comment:
'mobilenet_v2'

Reply:
Or 'mobilenet-v2'?

Reply:
I willl modify it 'mobilenet-v2'

Reply:
Both YOLO and DeepLab use underscore to separate its name and version.  So this should be `mobilenet_v2`.

Reply:
I see. I will modify it 'mobilenet_v2'.